src/style.css

```diff
@@ -0,0 +1,191 @@
+:root {
+  --color-white: #fff;
+  --color-grey: #616070;
+  --color-black: #060b35;
+  --color-blue: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--color-black);
+}
+
+/* card */
+.card {
+  position: relative;
+  display: block;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: var(--color-white);
+}
+
+.card__img {
+  position: relative;
+  width: 160px;
+  height: 0;
```

Comment:
The height of .card__img should not be 0. Set the height to the desired size and remove padding-bottom.